### PR TITLE
Implement home page design

### LINKED ISSUE #106: Implement home page design
@HTTPArchive/developers 

Using the design in #46, implement the HTML/CSS/JS. For any assets that we don't have (specific fonts or images) use placeholders for now.

### PR DESCRIPTION
**Pull request by tanhengyeow**
Fixes #106

### CODE REVIEW

**Comment by tanhengyeow** [comment]
@rviscomi This is the initial draft of the home page 😄 

There are some things that I would like to discuss:
1. Font and font size
2. Hex code for colors (I am currently using a color picker for estimation)
3. SVGs
   -  For the dropdown menus (year and language switcher), we might need an arrow down SVG. What do you think?
4. Images
   - I screen captured the 2 images placed in the `Methodology` section but including the original image would be more ideal :)
   - For the language switcher, how should we maintain the assets of country flags? Briefly discussed before in the [issue](https://github.com/HTTPArchive/almanac.httparchive.org/issues/50#issuecomment-503991517) for language switcher too.

**Comment by rviscomi** [comment]
Seeing the design actually taking shape makes this feel like a huge step closer to launch. Thanks for taking the time to build this, @tanhengyeow!

> Font and font size

If you have Photoshop, you can get more info about the text/assets from this PSD: https://drive.google.com/drive/folders/1peVnmbF9lmCVmHXZXE4JiADmDh2VdM9t

The font used in the design is [Poppins](https://fonts.google.com/specimen/Poppins?selection.family=Poppins). The body text should be 16px. Not sure about heading sizes and line height but eyeball them for now and we can tweak these later if needed.

> Hex code for colors (I am currently using a color picker for estimation)

Color picker is fine for now, I'll ask the designer to give us more specific guidelines.

> SVGs

These should be exportable from the PSD. I'll ask about getting individual SVG assets.

> Images

Ditto.

The language switcher should definitely use images rather than emoji. For example, the emoji approach fails for me on Chrome Linux:
image

File: src/static/css/splash.css [hunk 1]
```diff
@@ -1,59 +1,230 @@
...
+	color: #444;
+	margin: 0;
+}
+
+h2 {
+	font-weight: inherit;
+}
+
+button {
+	background: #f7f879;;
```
Comment (rviscomi) [inline src/static/css/splash.css]: Implement hover and focus states. The yellow border button becomes a yellow background button and vice versa.

For example, from this style:
image

to this style:
image

File: src/templates/en/splash.html [hunk 2]
```diff
@@ -8,28 +8,170 @@
...
+        </div>
+      </div>
+      <button>
+        Read the R2D2C3P0 Chapter
+      </button>
+    </div>
+  </section>
+  <section class="contributors-container">
+    <div class="contributors">
+      <h1>Contributors:</h1>
```
Comment (rviscomi) [inline src/templates/en/splash.html]:
```suggestion
      <h1>Contributors</h1>
```

File: src/templates/en/splash.html [hunk 3]
```diff
@@ -8,28 +8,170 @@
...
+          <div>07/10</div>
+          <div>Lorem ipsum</div>
+        </div>
+      </div>
+      <button>
+        Read the R2D2C3P0 Chapter
+      </button>
+    </div>
+  </section>
+  <section class="contributors-container">
```
Comment (rviscomi) [inline src/templates/en/splash.html]: See if you can also add the big "91 people" text on the right.

image

**Comment by rviscomi** [comment]
@tanhengyeow any progress on this? 😊

**Comment by tanhengyeow** [comment]
I'm a little occupied this week so I'll probably pick this up this weekend or next week 😄

**Comment by rviscomi** [comment]
Hi @tanhengyeow, have you made any progress on this? If not I'd propose merging this into a secondary branch so others could iterate on it.

**Comment by tanhengyeow** [comment]
Hi @rviscomi, yup I'm able to find time to finish this up 😄 Unfortunately I don't have Photoshop so I'll need some help with grabbing the relevant assets.

Here's the list of relevant assets I'll need:
- [ ] image
- [ ] image / perhaps also the `ja` flag
- [ ] image
- [ ] image

- [ ] image

- [ ] image

- [ ] image

- [ ] image

- [ ] image

- [ ] image (background of this section)

**Comment by hyperpress** [comment]
I personally really like this. It's very much like I would have done it -- if I had any artistic talent whatsoever.

**Comment by rviscomi** [comment]
Logo byline: let's use text and css
Flags: https://www.flaticon.com/packs/countrys-flags
Colorful rectangles: svg/css
Down icon and GitHub icon: https://shared-assets.adobe.com/link/44a4899f-53d1-4340-4f28-2eb282e2dc85
Header background: svg/css

Some of the graphics might be tricky in svg/css but I think it's preferable to adding more images, plus it gives us more control over responsiveness.

I'll work on getting you the rest.

**Comment by rviscomi** [comment]
Other assets are available here: https://shared-assets.adobe.com/link/44a4899f-53d1-4340-4f28-2eb282e2dc85

**Comment by rviscomi** [comment]
@tanhengyeow I'm going to merge what you have so far and we can iterate on it in follow-up PRs. This will also help unblock #130 and #154 so we can reuse styles.

**Pull request merged**